src/app.tsx

```diff
@@ -32,58 +21,41 @@ export const useStyles = makeStyles(theme => ({
     flexGrow: 1,
     padding: theme.spacing(3),
   },
+  close: {
+    padding: theme.spacing(0.5),
+  },
 }))
 /**
  * The main entry point of your app. You can start h@cking from here ;)
  */
 export const App: React.FunctionComponent = () => {
   const repo = useRepository()
   const [data, setData] = useState<Image[]>([])
-  const [uploaddata, setUploaddata] = useState<boolean>(false)
-  const [selectedimage, setSelectedimage] = React.useState<SelectedImage>({
-    imgIndex: 0,
-    imgPath: '',
-    imgTitle: '',
-    imgDescription: '',
-    imgAuthor: '',
-    imgAuthorAvatar: '',
-    imgCreationDate: '',
-    imgSize: '',
-    imgDownloadUrl: '',
-  })
+  const classes = useStyles()
+  const [isNotificationShown, ShowNotification] = React.useState<boolean>(false)
+  const [uploaddata, setUploaddata] = React.useState<boolean>(false)
```

Comment:
```suggestion
  const [isDataUploaded, setIsDataUploaded] = React.useState(false)
```
Please use camelCase for naming variables. Also it is a good pratice to prefix boolean variables with an _is_. This way you can see that this value is a boolean.

Reply:
done